src/widgets/conditions/conditions-overview.component.tsx

```diff
@@ -132,34 +132,28 @@ const ConditionsOverview: React.FC<ConditionsOverviewProps> = () => {
                         ))}
                       </TableRow>
                     ))}
-                    {!showAllConditions &&
-                      conditions?.length > conditionsToShowCount && (
-                        <TableRow>
-                          <TableCell colSpan={4}>
-                            <span
-                              style={{
-                                display: "inline-block",
-                                margin: "0.45rem 0rem"
-                              }}
-                            >
-                              {`${conditionsToShowCount} / ${conditions.length}`}{" "}
-                              {t("items", "items")}
-                            </span>
-                            <Button
-                              size="small"
-                              kind="ghost"
-                              onClick={toggleShowAllConditions}
-                            >
-                              {t("seeAll", "See all")}
-                            </Button>
-                          </TableCell>
-                        </TableRow>
-                      )}
                   </TableBody>
                 </Table>
               )}
             </DataTable>
           </TableContainer>
+          {totalRows > conditionsToShowCount && (
+            <Pagination
+              totalItems={conditions.length}
+              backwardText="Previous page"
```

Comment:
Translations for these strings would be superb.

Reply:
Sure, I will make the changes @FlorianRappl .